aea/protocols/base.py

```diff
@@ -33,6 +33,8 @@
 class Message:
     """This class implements a message."""
 
+    protocol_id = 'base'
```

Comment:
does that suggest we could also support the serialization for the `base` protocol? 
E.g. we could recommend using `JSONSerializer` to serialize objects of type `Message`.

Reply:
No, we don't support that at the moment. But we need to provide a class variable in the base class for typing consistency.